resources/Materials/TestSuite/_options.mtlx

```diff
@@ -111,6 +111,9 @@
 
     <!-- External testing: extra comma separated list of test root paths -->
     <input name="externalTestPaths" type="string" value="" />
+     
+    <!-- Color management configuration file. Only applicable if building with OCIO -->
+    <input name="colorManagementConfigFile" type="string" value="" />
```

Comment:
An explicit config file is required unit tests to run. Otherwise the default CM system is used.